src/modules/map/components/openlayers/OpenLayersMap.vue

```diff
@@ -357,17 +362,16 @@ export default {
         this.map.on('pointerdown', this.onMapPointerDown)
         // TODO: trigger a click after pointer is down at (roughly) the same spot
         // for longer than 1sec (no need to wait for the user to stop the click)
-        this.map.on('pointerup', this.onMapPointerUp)
+        // this.map.on('pointerup', this.onMapPointerUp) todo onMapPointerUp undefined
```

Comment:
What is the issue here?

Reply:
`onMapPointerUp` function not exists but I didn't add this code so I just comment it with todo